app/components/Carousel/components/ImageSubTitle/ImageSubTitle.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Image, View } from 'react-native';
+
+import { ExtendedText } from '../../../ExtendedText';
+import { IImageSubTitleProps } from './ImageSubTitle.props';
+
+export const ImageSubTitle: React.FC<IImageSubTitleProps> = ({ data }) => {
+  return (
+    <View>
+      <ExtendedText>{data.subTitle}</ExtendedText>
+      <Image source={data.image} />
```

Comment:
how can we adjust the size of image? 
please provide a way to configure the image size

Reply:
not now